.circleci/config.yml

```diff
@@ -61,6 +62,20 @@ workflows:
           filters:
             tags:
               only: /.*/
+      - test-render-linux-firefox:
```

Comment:
I live this new naming structure for the render test, should we also rename the old render test to adopt this naming structure? So that they all have the following `render-test-platform-browser-buildvariant`.

 Also, are these new tests prod/dev builds? We should hint that if we know

Reply:
Good idea, it will mean that the tests names will be longer but I think the advantage of clarity outweighs that.

Reply:
> Also, are these new tests prod/dev builds? We should hint that if we know

The new tests are dev builds, do you think we need to specify that? I'm currently leaning toward leaving it out to be more concise eg. `test-render` would be come `test-render-linux-chrome` instead of `test-render-linux-chrome-dev`. My current understanding of the prod + csp builds is that they have a narrow use case and it's unlikely that we would need to test them on multiple browsers/OS's.

Reply:
The new tests are dev builds, do you think we need to specify that? I'm currently leaning toward leaving it out to be more concise eg. test-render would be come test-render-linux-chrome instead of test-render-linux-chrome-dev.

I think that's ok, it's not worth trying to save a few characters :)
That also gives us clarity of what category of build we're dealing with when looking at test results. It was ok with a few builds, a little less so with 6+ render test runs.

Reply:
Sounds good, I added the `-dev` suffix to make that syntax consistent across all tests.